src/CommunityToolkit.Maui/Extensions/ValueConverterExtension.shared.cs

```diff
@@ -75,9 +75,9 @@ static bool IsValidNullableValueType(Type targetType)
 	private protected static TValue ConvertValue<TValue>(object? value) => value switch
 	{
 		null when IsNullable<TValue>() => default,
-		null when !IsNullable<TValue>() => throw new ArgumentNullException(nameof(value), $"Value cannot be null because {nameof(TValue)} is not nullable"),
+		null when !IsNullable<TValue>() => throw new ArgumentNullException(nameof(value), $"CurrentRating cannot be null because {nameof(TValue)} is not nullable"),
 		TValue convertedValue => convertedValue,
-		_ => throw new ArgumentException($"Value needs to be of type {typeof(TValue)}", nameof(value))
+		_ => throw new ArgumentException($"CurrentRating needs to be of type {typeof(TValue)}", nameof(value))
```

Comment:
Looks like a Find + Replace typo
```suggestion
		_ => throw new ArgumentException($"Value needs to be of type {typeof(TValue)}", nameof(value))
```